Fix unknown component status after rolling update of component operator deployment (#82)
cloudogu/k8s-component-operator: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

> ### Fixed
> - [#81] Components' status was `unknown` after rolling update of component operator deployment
> ### Changed
> - [#81] When we detect a change in the component operator deployment, the status of *ALL* components is updated, instead of just that of the component operator.

Reviewer: ```suggestion
### Fixed
- [#81] Components' status after rolling update of component operator deployment is not `unknown` any more
   - When we detect a change in the component operator deployment, the status of *ALL* components is updated, instead of just that of the component operator.
```
